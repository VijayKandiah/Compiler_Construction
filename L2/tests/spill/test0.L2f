(:myF
  0 0

  %myVar <- 0

  rdi <- %myVar
  call :myF2 0

  return
)

%myVar

%S
