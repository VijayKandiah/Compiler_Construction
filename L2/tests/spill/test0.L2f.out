(:myF
	0 1
	%S0 <- 0
	mem rsp 0 <- %S0
	%S1 <- mem rsp 0
	rdi <- %S1
	call :myF2 0
	return
)
